wgsl/index.bs

```diff
@@ -10383,14 +10396,14 @@ When several edges have to be created we use `X -> {Y, Z}` as a short-hand for `
 
       For each [=address spaces/function=] address space pointer parameter *i*,
       [=Value_return_i_contents=] -> *Vin*(*prev*) (see [[#func-var-value-analysis]])
-  <tr><td class="nowrap">*e2* = *e1*;
+  <tr><td class="nowrap">*e1* = *e2*;
```

Comment:
This matches https://dawn-review.googlesource.com/c/dawn/+/123120/4/src/tint/resolver/uniformity.cc#553 
which was independently developed.